src/compas_blender/artists/robotmodelartist.py

```diff
@@ -20,15 +20,17 @@ class RobotModelArtist(BaseRobotModelArtist):
         The name of the layer that will contain the robot meshes.
     """
 
-    def __init__(self, model, layer=None):
-        self.layer = layer
+    def __init__(self, model, collection=None, layer=None):
+        self.view_layer = layer
+        self.collection = collection
```

Comment:
the term `layer` was introduced here to be consistent with Rhino. internally it is supposed to be used as the name for the collection. if that is confusing we should change this to `collection` for all blender artists...

Reply:
actually i need to take that back. 

it seems we already removed this from the other artist in favor of implicitly named collections. perhaps this would be a good time to decide on a global strategy

Reply:
I think it makes more sense to use `collection` and that `layer` should be removed from `RobotModelArtist`.

Reply:
It would be great if (in the future) one could port a script from Rhino to blender just by changing the import statement, this would only work if all Robotmodelartists use the same parameter names. However using the word "layer" in the Blender context is generally confusing.

Reply:
agreed, but perhaps the solution to this is that we take a step back and use our own conventions, rather than transplanting the conventions from one system onto another. but i imagine that will be part of a lengthy discussion :)